client/my-sites/plugins/plugins-browser-item/style.scss

```diff
@@ -123,6 +153,12 @@
 	}
 }
 
+.plugins-browser-item__footer {
+	margin-top: auto;
+	display: flex;
+	justify-content: space-between;
+}
+
```

Comment:
We should also remove the `UpgradeButton` CSS

Reply:
done 8284966f3a808426e9a78433dd75c7ee74df16d7